drivers/opencl/src/main/java/uk/ac/manchester/tornado/drivers/opencl/graal/compiler/OCLCompilationResultBuilder.java

```diff
@@ -264,6 +265,33 @@ void emitLoopBlock(Block block) {
         emitBlock(block);
     }
 
+    void emitSkippedInstructions(Block block) {
```

Comment:
Document the need for "Skip" instructions. It looks to me more like a NOOP instruction.